package/yast2-snapper.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Mar 05 17:58:48 CET 2019 - aschnell@suse.com
+
+- enable rubocop
```

Comment:
I fear some check script will complain about not having a Bugzilla, Fate or Jira number.

Reply:
Yes, but there will be more changes soon.